build-scripts/graal.js

```diff
@@ -46,41 +46,44 @@ if (!fs.existsSync(TEMP_PATH)) {
   fs.mkdirSync(TEMP_PATH);
 }
 
-const NATIVE_IMAGE_BUILD_ARGS = [ '-H:+JNI'].concat(GRAAL_OS !== 'windows' ? ['--no-server'] : []).concat([
+const NATIVE_IMAGE_BUILD_ARGS = [
+  '-H:+JNI',
+  '--no-server',
   '-H:+ReportUnsupportedElementsAtRuntime',
   '-H:IncludeResourceBundles=com.google.javascript.rhino.Messages',
   '-H:IncludeResourceBundles=org.kohsuke.args4j.Messages',
   '-H:IncludeResourceBundles=org.kohsuke.args4j.spi.Messages',
   '-H:IncludeResourceBundles=com.google.javascript.jscomp.parsing.ParserConfig',
   `-H:ReflectionConfigurationFiles=${path.resolve(__dirname, 'reflection-config.json')}`,
-  '-H:IncludeResources=(externs.zip)|(.*(js|txt))'.replace(/\|/g, () => {
+  '-H:IncludeResources=(externs.zip)|(.*(js|txt))'.replace(/[\|\(\)]/g, (match) => {
```

Comment:
Can you add a comment here as to what this code is attempting to do?

Reply:
The comment [just below this](https://github.com/google/closure-compiler-npm/pull/174/files/050e882a5409b9543754deea7248d1b3da8b5a36#diff-0a1cc5f9b2cbccfcc18b3fd7db39dc45L58) was designed to do that. Can you specify what's unclear from that?

It's all windows shell escaping requirements and truthfully it requires a lot of trial and error just to figure out how to make it work.

Reply:
ok, good enough